test/browser/homepage.test.js

```diff
@@ -2,6 +2,17 @@ jest.setTimeout(10000);
 
 const URL = 'http://localhost:4444';
 
+/**
+ * Filters out external resources and duplicate coverage information
+ */
+function getUniqueResources(arr) {
```

Comment:
wow, good to know. I'm guessing this is caused by preloads? yikes.

Reply:
Yeah not sure why they are accessed twice. It's something we should keep an eye on.